community/kernel/src/main/java/org/neo4j/kernel/impl/api/TokenAccess.java

```diff
@@ -143,7 +143,11 @@ private static boolean hasAny( Iterator<?> iter )
         @Override
         public void close()
         {
-            statement.close();
```

Comment:
Is this strictly necessary or just being over the top safe?

Reply:
This was back-ported from the 3.3 branch. Not sure - what do you think, @henriknyman?

Reply:
Hmm I don't remember.. it could be the latter.